packages/documentation/src/pages/architecture/unit-tests.mdx

```diff
@@ -0,0 +1,120 @@
+## How do unit tests work?
+
+#### Where they live
+- Tests can live anywhere in the `src` directory, but in general are located in a directory close to the code or application that they are written to test.
+- The `.unit.spec.js` file extension is used for unit tests, while `.e2e.spec.js` is used for end-to-end (E2E) tests.
+- Visual regression test config has its own directory under `src/platform/testing/visual-regression`
+- Test commands are specified in `package.json`, as well as in vets-website's `README.md`
+- Configuration files for nightwatch and sass-lint live under `config/`
+- Scripts for running nightwatch tests live under `script/`
+
+#### When they run/ how they run
+There are 5 times tests can run:
+
+  1. At any time during local development by running one of the npm scripts specified in `package.json`
```

Comment:
I think we should opt to have this section describe when the tests are run automatically. 

Might be easier to scan this if we turned it into a table 

x axis: locally- automatically (pre-commit), Jenkins- after create PR, Jenkins- before deploying to any environment 
y axis: JS linting, SASS linting, unit, e2e, code climate, accessibility